### DISCUSSION TITLE
Student pack

### DISCUSSION THREAD

**Discussion by anthonycailleau** [community/community#120726] [GitHub Education]
Hello, I have any problems with the validation, i don't reveive email in my student mail box...
![image](https://github.com/community/community/assets/135410178/460b2714-d3c3-4838-99dc-2b3564ec9960)

**Comment by tr41z** [comment]
You need to wait couple weeks until they validate it. Then, after that, you'll receive all benefits and email confirmation.

**NOTE: You need to provide your student email there in order to get accepted.**

**Comment by samus-aran** [comment]
Hi @anthonycailleau 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
